.github/workflows/vast.yaml

```diff
@@ -15,6 +15,9 @@ on:
     types:
       - opened
       - synchronize
+  merge_group:
+    types:
+      - checks_requested
```

Comment:
This also needs to be done for the analysis, blocked, and style-check workflow files.

Reply:
We talked about this quickly in the office, and agreed that it's just style check we need to add. I misunderstood the trigger.

Reply:
We agreed that `Tenzir` and `Style Check` are all we need.